camera_service/camera_service.py

```diff
@@ -0,0 +1,190 @@
+import os
+import asyncio
+import numpy as np
+from caproto.server import ioc_arg_parser, run, pvproperty, PVGroup
+import simulacrum
+import zmq
+import time
+from zmq.asyncio import Context
+import pickle
+
+class ProfMonService(simulacrum.Service):
+    default_image_dim = 1024
+    util_pvs = ['EVR:IN20:PM01:CTRL.DG0E', 'EVR:IN20:PM02:CTRL.DG1E', 'EVR:IN20:PM02:CTRL.DG0E', 'EVR:IN20:PM02:CTRL.DG0E',
```

Comment:
these look pretty easy to generate in a loop. something like
```
for i in range(1, 7):
    "EVR:IN20:PM0{PROFMON}:CTRL.DG0E".format(PROFMON=i)
    "EVR:IN20:PM0{PROFMON}:CTRL.DG1E".format(PROFMON=i)
```

Reply:
Some of them can be generated in a loop, but enough can't be that I think it might be easier to just have a hardcoded list that I can easily add/take out. I took out the repeats and it's a shorter list now.